src/constants.rs

```diff
@@ -23,12 +23,9 @@ pub const TABLE_GAP_HEIGHT_LIMIT: u16 = 7;
 pub const TIME_LABEL_HEIGHT_LIMIT: u16 = 7;
 
 // Side borders
-pub static SIDE_BORDERS: Lazy<tui::widgets::Borders> =
-    Lazy::new(|| tui::widgets::Borders::from_bits_truncate(20));
-pub static TOP_LEFT_RIGHT: Lazy<tui::widgets::Borders> =
-    Lazy::new(|| tui::widgets::Borders::from_bits_truncate(22));
-pub static BOTTOM_LEFT_RIGHT: Lazy<tui::widgets::Borders> =
-    Lazy::new(|| tui::widgets::Borders::from_bits_truncate(28));
+pub const SIDE_BORDERS: tui::widgets::Borders = tui::widgets::Borders::from_bits_truncate(20);
+pub const TOP_LEFT_RIGHT: tui::widgets::Borders = tui::widgets::Borders::from_bits_truncate(22);
+pub const BOTTOM_LEFT_RIGHT: tui::widgets::Borders = tui::widgets::Borders::from_bits_truncate(28);
```

Comment:
These 2 are unused.

Reply:
Hm, feel free to remove them if you want in this PR.